src/argilla/client/users.py

```diff
@@ -115,16 +114,21 @@ def __str__(self) -> str:
         )
 
     @staticmethod
-    def __active_client(httpx: bool = True) -> Union["httpx.Client", "AuthenticatedClient"]:
+    def __active_client() -> Union["httpx.Client", "AuthenticatedClient"]:
```

Comment:
As mentioned before, I wouldn't revert this until the mock is fixed in a separate PR

Reply:
rollback